ambassador/README.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: pattern
+title: Ambassador
+folder: ambassador
+permalink: /patterns/ambassador/
+categories: Structural
+tags:
+ - Java
+ - Difficulty-Intermediate
+---
+
+## Intent
+Provide a helper service that sends network requests on behalf of a client and offload common additional connectivity tasks.
```

Comment:
Isn't networking just one example of Ambassador usages? Could the description be more general?